### PR TITLE
Deployment edits

### PR DESCRIPTION
**Pull request by Jkovarik**
Minor update to account for markdown lint, add audit configuration.

### CODE REVIEW

File: docs/development/quality-and-coverage.md [hunk 1]
```diff
@@ -22,7 +22,19 @@ file.
...
 To check the configured linting, run `npm run lint`.
 
+## Documentation quality checking
+
+This project uses [markdownlint-cli](https://www.npmjs.com/package/markdownlint-cli)
+as a frontend to [markdownlint](https://www.npmjs.com/package/markdownlint) to check
+all of our markdown for style and formatting.   The configured rules can be found
+[here](https://github.com/nasa/cumulus/blob/master/.markdownlint.json).
+
+To check the configured markdown linting run `npm run lint-md`.
```
Comment (markdboyd) [inline docs/development/quality-and-coverage.md]:
```suggestion
To run the linting of markdown files, run `npm run lint-md`.
```

Comment (Jkovarik) [reply docs/development/quality-and-coverage.md]: @markdboyd  how about 

```suggestion
To run linting on the markdown files  run `npm run lint-md`.
```

File: docs/development/quality-and-coverage.md [hunk 2]
```diff
@@ -22,7 +22,19 @@ file.
...
 To check the configured linting, run `npm run lint`.
 
+## Documentation quality checking
+
+This project uses [markdownlint-cli](https://www.npmjs.com/package/markdownlint-cli)
+as a frontend to [markdownlint](https://www.npmjs.com/package/markdownlint) to check
+all of our markdown for style and formatting.   The configured rules can be found
+[here](https://github.com/nasa/cumulus/blob/master/.markdownlint.json).
+
+To run linting on the markdown files  run `npm run lint-md`.
```
Comment (markdboyd) [inline docs/development/quality-and-coverage.md]:
```suggestion
To run linting on the markdown files, run `npm run lint-md`.
```

Comment (markdboyd) [reply docs/development/quality-and-coverage.md]: Sorry, nit

**Pull request merged**